bin/smart_proxy_dynflow_core

```diff
@@ -3,5 +3,22 @@ require 'dynflow'
 require 'rack'
 require 'smart_proxy_dynflow_core/launcher'
 require 'yaml'
+require 'optparse'
 
-SmartProxyDynflowCore::Launcher.launch!
+options = {}
+OptionParser.new do |opts|
+  opts.on('-c', '--config-dir CONFIG_DIR', String, 'Directory to load settings from') do |value|
+    options[:config_dir] = value
+  end
+
+  opts.on('-1', '--one-config', String, 'Do not load more than 1 config') do |value|
```

Comment:
Since it by default traverses all the possible paths where the config could be and merges all the discovered configs I thought it would be nice to have a way to say "load the config from $place and nothing else"